lesson_05/homework_05.py

```diff
@@ -1,39 +1,135 @@
 small_list = [3, 1, 4, 5, 2, 5, 3]
 big_list = [3, 5, -2, -1, -3, 0, 1, 4, 5, 2]
 # task 1. Знайдіть всі унікальні елементи в списку small_list
+new_list = []
+for i in set(small_list):
+    if small_list.count(i) == 1:
+        new_list.append(i)
+print(f"Унікальні елементи списку small_list: {new_list}")
```

Comment:
значно простіше, якщо цикл викинути:
```suggestion
print(f"Унікальні елементи списку small_list: {set(big_list)}")
```